arangod/Cluster/HeartbeatThread.cpp

```diff
@@ -697,6 +707,8 @@ void HeartbeatThread::getNewsFromAgencyForCoordinator() {
   if (++_DBServerUpdateCounter >= 60) {
     ci.loadCurrentDBServers();
     _DBServerUpdateCounter = 0;
```

Comment:
The variable name `_DBServerUpdateCounter` is a bit misleading here, as it is obviously also used for coordinators. 
So we should probably rename it to `_updateIteration` or so.